src/core/radio/trel_interface.hpp

```diff
@@ -228,6 +228,22 @@ class Interface : public InstanceLocator
      */
     bool IsFilterEnabled(void) const { return mFiltered; }
 
+    /**
+     * Gets the TREL counters.
+     *
+     * The counters are initialized to zero when the OpenThread instance is initialized.
+     *
+     */
+    const otTrelCounters *GetTrelCounters(void) const { return &mTrelCounters; }
+
+    /**
+     * Returns the number of TREL peers.
+     *
+     * @returns  The number of TREL peers.
+     *
+     */
+    uint16_t GetTrelPeersNum(void) const { return mPeerTable.GetLength(); }
```

Comment:
Rename to `GetNumberOfPeers()`?
Also move the definition after `const Peer *GetNextPeer(PeerIterator &aIterator) const;` so all method dealing with `Peer` list to be closer to each other.

Reply:
Done.